zerver/tests/test_import_export.py

```diff
@@ -1012,6 +1016,12 @@ def assert_realm_values(f: Callable[[Realm], object]) -> None:
             f'<p><span class="user-mention" data-user-id="{imported_polonius_user.id}">@Polonius</span></p>',
         )
 
+        imported_hamlet_user = UserProfile.objects.get(
+            delivery_email=self.example_email("hamlet"), realm=imported_realm
+        )
+        imported_denmark_stream = Stream.objects.get(name="Denmark", realm=imported_realm)
+        self.assertEqual(imported_denmark_stream.creator, imported_hamlet_user)
```

Comment:
Apparently this doesn't actually fail without the fix above; because tests run inside a transaction. Not sure how to simulate the real situation in this test reasonably... But verified manually that the original code fails to import if there's a stream with `.creator` set, and after this fix it's able to be imported correctly.

Reply:
Probably that's good enough.

Reply:
#30082 fails(error) without this fix.